tools/Targets/Microsoft.Spot.system.gcc.targets

```diff
@@ -52,7 +52,7 @@
     <CC_CPP_TARGETTYPE_FLAGS Condition="'$(FLAVOR)'=='Instrumented'" >$(CC_CPP_TARGETTYPE_FLAGS) -O0</CC_CPP_TARGETTYPE_FLAGS>
     <CC_CPP_TARGETTYPE_FLAGS Condition="'$(FLAVOR)'=='Release'"      >$(CC_CPP_TARGETTYPE_FLAGS) -Os</CC_CPP_TARGETTYPE_FLAGS>
 
-    <CC_FLAGS>$(CC_FLAGS)-xc</CC_FLAGS>
+    <CC_FLAGS>$(CC_FLAGS)-std=c11 -xc</CC_FLAGS>

```

Comment:
This is a global change impacting ALL solutions building with GCC, thus needs some special care so we don't break working builds arbitrarily. The settings added should be conditional (e.g. opt-in from the project)